.travis.yml

```diff
@@ -0,0 +1,29 @@
+# This config file for Travis CI
+sudo: required
+dist: trusty
+services:
+  - docker
+language: cpp
+cache: ccache
+compiler: gcc
+
+notifications:
+  email:
+    recipients:
+      - levi.armstrong@swri.org
+env:
+  global:
+    - ROS_DISTRO=kinetic
+    - UPSTREAM_WORKSPACE=file
+    - ROSINSTALL_FILENAME=dependencies.rosinstall
+    - TEST_BLACKLIST=bullet3_ros,fcl_ros,libccd_ros
+    - MAKEFLAGS="-j 4"
+    - NOT_TEST_INSTALL=true
+  matrix:
+    - ROS_DISTRO=kinetic ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu
+    - ROS_DISTRO=kinetic ROS_REPOSITORY_PATH=http://packages.ros.org/ros-shadow-fixed/ubuntu
+    - ROS_DISTRO=kinetic ROS_REPO=ros TEST=clang-format
```

Comment:
where did you find `TEST=clang-format`? The clang format PR is still pending and for now we do not support `TEST=`.

Reply:
I believe it was from the moveit repository. I will look.

Reply:
`moveit` isn't using `industrial_ci`.

Reply:
Ok, that make sense why clang never gets run.

Reply:
I removed it.